packages/itinerary-body/src/ItineraryBody/accessibility-rating.tsx

```diff
@@ -0,0 +1,90 @@
+import React from "react";
+import styled from "styled-components";
+import { Wheelchair } from "@styled-icons/foundation/Wheelchair";
+
+interface WrapperProps {
+  color: string;
+  large: boolean;
+}
+
+const Wrapper = styled.div<WrapperProps>`
+  align-items: center;
+  background-color: ${props => props.color};
+  border-radius: ${props => (props.large ? "4px" : "20px")};
```

Comment:
Isn't that supposed to be the other way? (large should get the larger radius?)

Reply:
The large view is used beneath the mode icons which have a similar radius so this is intentional.